src/main/scala/cromwell/engine/SymbolStore.scala

```diff
@@ -26,9 +26,50 @@ class SymbolStore(binding: WdlBinding, inputs: Map[FullyQualifiedName, WdlValue]
     assignSymbolStoreEntry(fullyQualifiedName, value, input = true)
   }
 
+  for {
+    workflow <- binding.workflows
+    call <- workflow.calls
+    (k, v) <- call.inputMappings
+  } yield assignSymbolStoreEntry(s"${call.fullyQualifiedName}.$k", v, input = true)
+
+  private def callOutputEntries(call: Call): Seq[SymbolStoreEntry] =
+    store.filter {entry => entry.scope == call.fullyQualifiedName && entry.isOutput}.toSeq
+
+  private def callInputEntries(call: Call): Seq[SymbolStoreEntry] =
+    store.filter {entry => entry.scope == call.fullyQualifiedName && entry.isInput}.toSeq
+
   def locallyQualifiedInputs(call: Call): Map[String, WdlValue] = {
-    store.collect { case entry if entry.scope == call.fullyQualifiedName && entry.isInput =>
-      entry.key.name -> entry.wdlValue.get
+
+    def lookup(s: String): WdlValue = {
+      val workflow = call.parent.map {_.asInstanceOf[Workflow]} getOrElse {
+        throw new WdlExpressionException("Expecting 'call' to have a 'workflow' parent")
+      }
+      val matchedCall = workflow.calls.find {c => c.name == s}.getOrElse {
+        throw new WdlExpressionException(s"Expecting to find a call with name '$s'")
+      }
+      val callOutputs = callOutputEntries(matchedCall) map {entry =>
+        val value = entry.wdlValue match {
```

Comment:
It'd be good to take the closure into a function so you can just say `callOutputEntries(matchedCall) map doSomething` or something like that